libs/components/molecules/src/lib/footer/footer.component.ts

```diff
@@ -0,0 +1,86 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { ConnectedPosition, OverlayModule } from '@angular/cdk/overlay';
+import { MatListModule } from '@angular/material/list';
+
+/**
+ * Download format interface
+ */
+export interface DownloadFormat {
+  /**
+   * format label
+   */
+  label: string;
+}
+
+/**
+ * cdk panel list position
+ */
+const DOWNLOADS_LIST_POSITION: ConnectedPosition[] = [
+  {
+    panelClass: 'above',
+    originX: 'start',
+    originY: 'top',
+    overlayX: 'start',
+    overlayY: 'bottom',
+    offsetY: 0,
+  },
+  {
+    panelClass: 'below',
+    originX: 'start',
+    originY: 'bottom',
+    overlayX: 'start',
+    overlayY: 'top',
+  },
+];
+
+/**
+ * Component
+ */
+@Component({
+  selector: 'hra-footer',
+  standalone: true,
+  imports: [CommonModule, MatButtonModule, MatIconModule, OverlayModule, MatListModule],
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FooterComponent<T extends DownloadFormat = DownloadFormat> {
+  /**
+   * Input  of footer component
+   */
+  @Input() downloadFormats: T[] = [];
+
+  /**
+   * Output  of footer component
+   */
+  @Output() readonly download = new EventEmitter<T>();
+
+  /**
+   * Output  of Illustration
+   */
+  @Output() readonly IlustrationButtonClick = new EventEmitter();
```

Comment:
- Incorrect formating. Should start with lowercase and not have the word button in it i.e. `illustrationClick`
- Misspelled, correct spelling `illustration`
- Missing type for event emitter
- Same goes for all other outputs